build/Program.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
 using System.IO;
-using System.Linq;
 using static Bullseye.Targets;
 using static SimpleExec.Command;
 
 namespace build
 {
-    class Program
+    internal static class Program
```

Comment:
There was a mix of implicit (e.g. here) and explicit (e.g. `private static class Targets ...`) accessibility in this file, so I made everything explicit.